vars/kola.groovy

```diff
@@ -68,7 +68,7 @@ def call(params = [:]) {
             if (!params['skipBasicScenarios']) {
                 shwrap("cd ${cosaDir} && cosa kola run ${rerun} --output-dir=${outputDir}/kola --basic-qemu-scenarios")
             }
-            shwrap("cd ${cosaDir} && cosa kola run ${rerun} --output-dir=${outputDir}/kola --build=${buildID} ${arch} ${platformArgs} --tag '!reprovision' --parallel ${parallel} ${args} ${extraArgs}")
+            shwrap("cd ${cosaDir} && cosa kola run ${rerun} --output-dir=${outputDir}/kola --denylist-test basic --build=${buildID} ${arch} ${platformArgs} --tag '!reprovision' --parallel ${parallel} ${args} ${extraArgs}")
```

Comment:
so the user still wants to run the `basic` tests even if they pass `skipBasicScenarios: True`?

Reply:
I'd say so, because `--basic-qemu-scenarios` is about booting the QEMU image using the different QEMU firmware/NVMe options. It just uses the `basic` test as a convenient tiny target. It makes sense to want to skip that, but it doesn't imply you want to skip [the actual checks in the `basic` test](https://github.com/coreos/coreos-assembler/blob/93efb63dcbd63dc04a782e2c6c617ae0cd4a51c8/mantle/kola/tests/coretest/core.go#L48-L55) which are unrelated to the QEMU stuff.

Reply:
I guess simplest would be to just drop that commit for now?

Reply:
ok i'll drop this commit